javascript-modal/main.js

```diff
@@ -0,0 +1,24 @@
+var openModal = document.querySelector('.open-modal');
+var modalFull = document.querySelector('.modal-full');
+var modalWindow = document.querySelector('.column');
+var isOff = (modalFull.className === 'modal-full off');
+var button = document.querySelector('.modal-no');
+var text = document.querySelector('.text-no');
+
+openModal.addEventListener('click', function (event) {
+  if (isOff) {
```

Comment:
Doesn't look like you ever change the value of `isOff` here, you can likely remove the variable and this conditional.